src/simularium/VisGeometry.ts

```diff
@@ -370,14 +388,67 @@ class VisGeometry {
         } else {
             this.resetBounds(DEFAULT_VOLUME_DIMENSIONS);
         }
-        if (this.resetCameraOnNewScene) {
-            this.resetCamera();
+
+        // Get default camera transform values from data
+        if (trajectoryFileInfo.cameraDefault) {
+            this.cameraDefault = trajectoryFileInfo.cameraDefault;
+        } else {
+            this.logger.warn(
```

Comment:
this isn't really flipping back to the default, right?  It's just leaving the most recent camera settings?

Reply:
Ohhh good catch! I'll fix it

Reply:
could set this.cameraDefault = DEFAULT_CAMERA_SPEC here, if that would help